src/vos/vos_io.c

```diff
@@ -195,7 +195,8 @@ vos_ioc_destroy(struct vos_io_context *ioc, bool evict)
 	vos_ilog_fetch_finish(&ioc->ic_dkey_info);
 	vos_ilog_fetch_finish(&ioc->ic_akey_info);
 	vos_cont_decref(ioc->ic_cont);
-	vos_ts_set_free(ioc->ic_ts_set);
+	if (ioc->ic_ts_set)
```

Comment:
is it needed?   We've removed such checks in many places

Reply:
This is another case where we're seeing 100s of log lines.  They were initially all reported in the header file but as it's a one-line function I made it a macro and they all originate here.